latest_cov_build.py

```diff
@@ -0,0 +1,26 @@
+import os
+import tarfile
+
+import taskcluster
+
+try:
+    from urllib.request import urlretrieve
+except ImportError:
+    from urllib import urlretrieve
+
+
+# Retrieving task from index
+index = taskcluster.Index()
+taskId = index.findTask('gecko.v2.mozilla-central.' +
+                        'latest.firefox.linux64-ccov-opt')['taskId']
+# taskId uniquelly identifies artifact
```

Comment:
There's no need for that, we always want the latest build so we don't need to store the old ones too